workload_mapper/mariadb.rb

```diff
@@ -0,0 +1,6 @@
+mariadb_config = "/etc/my.cnf"
+if system.runs_service?("mysql") && system.has_file?(mariadb_config)
+  identify "mariadb", "db"
```

Comment:
Shouldn't the `"db"` rather be in the `mariadb.yml` file? This is describing the role of the workload. Is this an attribute of the inspected system or is it an attribute of the workload itself?